Maintainer.md

```diff
@@ -1,33 +1,34 @@
 ## Maintainer's tasks on package submission to SailfishOS:Chum
 
-On initial submission of a package, maintainers are expected to check that …
+On initial submission of a package by a package maintainer ("submitter" hereafter), a SailfishOS:Chum
+maintainer is tasked to check that …
 
-- `_service` file contains a specific git tag or commit ID (hash).
+- the `_service` file contains a specific git tag or commit ID (hash).
   Alternative is source submission via compressed archive.
 
-- `_service` file does not contain a webhook.
+- the `_service` file does not contain a webhook.
 
-- License in rpm spec file is [approved by OSI](https://opensource.org/licenses) and
+- the license in rpm spec file is [approved by OSI](https://opensource.org/licenses) and
   matches the software license in the source code repository or the submitted archive.
   
-- License tag in rpm spec file is one of
+- the license tag in rpm spec file is one of
   [github.com/sailfishos/rpmlint/blob/master/rpm/sailfish.toml#L150-L459](https://github.com/sailfishos/rpmlint/blob/master/rpm/sailfish.toml#L150-L459).
 
-- No binary blobs, such as precompiled libraries, applications or other executable files, are part of the package submission.
+- no binary blobs, such as precompiled libraries, applications or other executable files, are part of the package submission.
   Bundled databases or similar are allowed.
   
-- Package does not conflict with (e.g., replace) any package from Jolla's repositories for Sailfish&nbsp;OS.
+- the package does not conflict with (e.g., replace) any package from Jolla's repositories for Sailfish&nbsp;OS.
 
 After these checks have been successful, …
 
 - accept the submission of the package to `sailfishos:chum:testing`.
 
-- make the submitter a maintainer for the package in `sailfishos:chum:testing`.
+- employ the submitter as a maintainer for the package in `sailfishos:chum:testing`.
 
 - if building the package fails for some versions of SFOS, decide together with the submitter
   whether to disable those build targets or to adjust the package.
 
-- with all compilation issues resolved, promote the package from `sailfishos:chum:testing`
+- and all compilation issues are resolved, promote the package from `sailfishos:chum:testing`
```

Comment:
```suggestion
- after all compilation issues are resolved, promote the package from `sailfishos:chum:testing`
```

Reply:
Oh, a classic!
The real issue is a sentence which spans across bullet points, which is a no-go, because bullet points are "stronger" (i.e., of higher priority) than any punctuation.  Thus a bullet point can contain a word, a phrase, one or multiple sentences, but a sentence cannot span across multiple bullet points.  But people love it and do that in all languages I know at least a bit.

There are two variants of this no-go, one is a single sentence "flowing" across all bullet points (then one can simply drop all bullet points to obtain a proper sentence), the other is an introductory part of a sentence at the start, with multiple endings after each bullet point.

Specifically with variant two (e.g., here), it is very easy for writers (and often also for readers) to loose track of the introductory half-sentence, so that the concluding part after a bullet point does not really fit (WRT grammar, tenses singular vs. plural etc.).

* The original is: *"After these checks have been successful, with all compilation issues resolved, promote the package from `sailfishos:chum:testing` to `sailfishos:chum`."*
* My version is: *"After these checks have been successful, and all compilation issues are resolved, promote the package from `sailfishos:chum:testing` to `sailfishos:chum`."*
* Your version is: *"After these checks have been successful, after all compilation issues are resolved, promote the package from `sailfishos:chum:testing` to `sailfishos:chum`."*

IMO, on a logical level,  "after the checks are passed" *and* "the compilation issues are resolved", not "after the checks are passed" *after* "the compilation issues are resolved".   Furthermore the double "after" sounds really strange.

But without the "After" at the start, the other endings do not fit.

And all in all this is trying to fix a fundamentally wrong construct: A sentence which spans across bullet points.

I do not mind: After rereading it multiple times I think the original variant was slightly better than yours or mine, but a "while" with the "are" is least awkward (and at least acceptable WRT logic).

P.S.: The proper solution would be to re-phrase the construct with a whole introductory sentence ending with a colon ("**:**"), and let each bullet point contain a whole sentence ending with a full stop ("**.**").

Reply:
No, it is not. This was pretty much made as a list for us from the tasks that we have to do. Each bullet point can be separate full phrase if you wish. Swapping "with" -> "while" changes a meaning for me. Before it stated that I will make an action specified in the second part after the condition is met, now I am expected it to do "while".

Reply:
Not that has not been and is not the case, because "resolved" is past tense.  Hence it must be a precondition, not a task to fulfil.

But I don't care, so I went back to "with" (and thus no "are") by [commit 70afff9](https://github.com/sailfishos-chum/main/pull/67/commits/70afff9406750cc8d21f711fadab9cfd6e222930).

Reply:
Thank you. LGTM